contentrouting/delegated/http.go

```diff
@@ -2,114 +2,50 @@ package delegated
 
 import (
 	"context"
-	"encoding/json"
-	"fmt"
-	"net/http"
 
+	finderhttpclient "github.com/filecoin-project/storetheindex/api/v0/finder/client/http"
```

Comment:
we should redefine these in the client. the goal is multiple different index implementations